app/routes/events/utils.ts

```diff
@@ -14,67 +14,92 @@ import type {
 } from 'app/models';
 import type { DetailedUser } from 'app/store/models/User';
 
-// Current eventTypes
-export const EVENT_CONSTANTS = {
-  company_presentation: 'Bedriftspresentasjon',
-  lunch_presentation: 'Lunsjpresentasjon',
-  alternative_presentation: 'Alternativ bedpres',
-  course: 'Kurs',
-  breakfast_talk: 'Frokostforedrag',
-  party: 'Fest',
-  social: 'Sosialt',
-  event: 'Arrangement',
-  kid_event: 'KiD-arrangement',
-  other: 'Annet',
-} as const;
+export type ConfigProperties = {
+  displayName: string;
+  color: string;
+  textColor: string;
+};
+
+export const EventTypeConfig: Record<EventType, ConfigProperties> = {
+  [EventType.COMPANY_PRESENTATION]: {
+    displayName: 'Bedriftspresentasjon',
+    color: '#A1C34A',
+    textColor: '#000',
+  },
+  [EventType.COURSE]: {
+    displayName: 'Kurs',
+    color: '#52B0EC',
+    textColor: '#000',
+  },
+  [EventType.PARTY]: {
+    displayName: 'Fest',
+    color: '#FCD748',
+    textColor: '#000',
+  },
+  [EventType.SOCIAL]: {
+    displayName: 'Sosialt',
+    color: 'var(--color-event-red)',
+    textColor: '#FFF',
+  },
+  [EventType.BREAKFAST_TALK]: {
+    displayName: 'Frokostforedrag',
+    color: '#86D1D0',
+    textColor: '#000',
+  },
+  [EventType.LUNCH_PRESENTATION]: {
+    displayName: 'Lunsjpresentasjon',
+    color: '#A1C34A',
+    textColor: '#000',
+  },
+  [EventType.EVENT]: {
+    displayName: 'Arrangement',
+    color: 'var(--color-event-red)',
+    textColor: '#FFF',
+  },
+  [EventType.ALTERNATIVE_PRESENTATION]: {
+    displayName: 'Alternativ bedpres',
+    color: '#8A2BE2',
+    textColor: '#FFF',
+  },
+  [EventType.KiD_EVENT]: {
+    displayName: 'KiD-arrangement',
+    color: 'var(--color-event-black)',
+    textColor: '#FFF',
+  },
+  [EventType.OTHER]: {
+    displayName: 'Annet',
+    color: 'var(--color-event-black)',
+    textColor: '#FFF',
+  },
+};
 
 // Returns the string representation of an EventType
-export const eventTypeToString = (eventType: EventType): string => {
-  return EVENT_CONSTANTS[eventType] || EVENT_CONSTANTS['other'];
+export const displayNameForEvent = (eventType: EventType) => {
+  return (
+    EventTypeConfig[eventType]?.displayName ||
+    EventTypeConfig[EventType.OTHER].displayName
+  );
```

Comment:
imo this would make a lot more sense if it's renamed to `displayNameForEventType`

Reply:
Yes completely agree. I'll change the other ones as well